a_tour_of_textile.yml

```diff
@@ -0,0 +1,472 @@
+examples:
+  start_a_chat:
+    add_thread:
+      cmd:
+        code: 'textile threads add "Chat" --type="open" --sharing="shared" --whitelist="P7X3gZus5H15tWCxk4oP6EVsgAM9vwUfCyepAKw49QuRyPYs"'
+      js_http_client:
+        code: 'Textile.threads.add("chat", undefined, undefined, "open", "shared", ["P7X3gZus5H15tWCxk4oP6EVsgAM9vwUfCyepAKw49QuRyPYs"])'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    invite_create:
+      cmd:
+        code: 'textile invites create --thread="12D3KooWExn4ut4RV2qHXFSiWb3AfhL2whB8vJgYpnDcmVCG7UBv"'
+      js_http_client:
+        code: 'Textile.invites.create("12D3KooWExn4ut4RV2qHXFSiWb3AfhL2whB8vJgYpnDcmVCG7UBv")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    accept_invite:
+      cmd:
+        code: 'textile invites accept "QmYzhyFhRGX3GBgsLMKoGrQqMWwPFKyPtsGmGbvma63zCf" --key="cCBPKRN6723KkroCfMsLVHj3cbVkwpg47s5wdjyEPxXz6rRoo6mjBZqiizd"'
+      js_http_client:
+        code: 'Textile.invites.accept("QmYzhyFhRGX3GBgsLMKoGrQqMWwPFKyPtsGmGbvma63zCf", "cCBPKRN6723KkroCfMsLVHj3cbVkwpg47s5wdjyEPxXz6rRoo6mjBZqiizd")'
+      react_native:
+        code: 'TODO'
+      ios:
+        code: 'TODO'
+      android:
+        code: 'TODO'
+    join_chat:
+      cmd:
+        code: 'textile chat --thread="12D3KooWExn4ut4RV2qHXFSiWb3AfhL2whB8vJgYpnDcmVCG7UBv"'
+      js_http_client:
+        code: '/* interactive chat is only available via CMD */'
+      react_native:
+        code: '/* interactive chat is only available via CMD */'
+      ios:
+        code: '/* interactive chat is only available via CMD */'
+      android:
+        code: '/* interactive chat is only available via CMD */'
+    join_chat_peer:
+      cmd:
+        code: 'textile chat --thread="12D3KooWExn4ut4RV2qHXFSiWb3AfhL2whB8vJgYpnDcmVCG7UBv" --api="http://localhost:41600"'
+      js_http_client:
+        code: '/* interactive chat is only available via CMD */'
+      react_native:
+        code: '/* interactive chat is only available via CMD */'
+      ios:
+        code: '/* interactive chat is only available via CMD */'
+      android:
+        code: '/* interactive chat is only available via CMD */'
+  my_runs:
+    add_thread:
+      cmd:
+        code: 'textile threads add "My runs" --schema-file "~/Downloads/location.json" --type="public" --sharing="invite_only"'
+      js_http_client:
+        code: 'Textile.threads.add("My runs", JSON.stringify(locations), undefined, "public", "invite_only")'
```

Comment:
We don't yet support arbitrary schemas in `js-http-client`, we could add this though. Currently, you'd need to add the schema first and reference it by its hash. Also, should be no reason to use `JSON. stringify` here when we do... should just take raw JSON and do the right thing in the background.